examples/js/loaders/FBXLoader.js

```diff
@@ -872,9 +875,10 @@
 			}
 
 			if ( uvInfo ) {
-
-				vertex.uv.fromArray( getData( polygonVertexIndex, polygonIndex, vertexIndex, uvInfo ) );
-
+				for (var i =0; i < uvInfo.length; i++){
```

Comment:
It would be great if you regard the code style of this file. Use can use this website for verification: http://zz85.github.io/mrdoobapproves/

Reply:
@Mugen87 thanks for the guide, I pushed the style change, let me know if it looks ok!